pkgs/applications/version-management/gitkraken/default.nix

```diff
@@ -85,7 +85,9 @@ stdenv.mkDerivation rec {
     rm -rf bin/gitkraken share/lintian
     cp -av share bin $out/
     popd
-    ln -s $out/share/gitkraken/gitkraken $out/bin/gitkraken
+
+    makeWrapper $out/share/gitkraken/gitkraken $out/bin/gitkraken \
+      --prefix XDG_DATA_DIRS : $GSETTINGS_SCHEMAS_PATH
```

Comment:
Please `gappsWrapperHook` instead of a manual `makeWrapper` invocation: 
https://github.com/NixOS/nixpkgs/pull/43150/files

Reply:
@Mic92 Thanks for the feedback. I've updated it.